pkg/tlsf/.gitattributes

```diff
@@ -1,2 +0,0 @@
-# Ignore whitespace errors inside these patch files by regarding them as binary.
-patch.txt binary
```

Comment:
(and this is not a hypothetical situation I construct here. I ran into this problem several times with packages.)